components/humantask/humantask-explorer-web/src/web/template/statsView.jag

```diff
@@ -0,0 +1,94 @@
+<div class="col-sm-12">
+  <div class="row">
+    <% if(readyTaskList.length() > 0 || completedTaskList.length() > 0 || reservedTaskList.length() > 0 ||
```

Comment:
Don't we need to handle these Lists becoming 'undefined' scenario..?Otherwise it could throw js errors when one of these lists become null/undefined.